packages/hooks/src/useCFDCanGetMoreMT5Accounts.ts

```diff
@@ -0,0 +1,36 @@
+import { useStore } from '@deriv/stores';
+import { CFD_PLATFORMS } from '@deriv/shared';
+import useIsClientHighRiskForMT5 from './useIsClientHighRiskForMT5';
+
+const ACCOUNT_TYPES = ['synthetic', 'financial', 'all'] as const;
+
+/**
+ * Custom hook to determine whether a client is eligible to get more MT5 CFD accounts.
+ * @returns `true` if the client is eligible to get more MT5 CFD accounts, false otherwise.
+ */
+const useCFDCanGetMoreMT5Accounts = () => {
```

Comment:
useCFDCanGetMoreMT5Accounts, can we rename to useCanGetMoreMT5Accounts

Reply:
I suggest keeping it with CFD as an identifier like we've in other hooks